language-support/java/bindings-rxjava/src/test/scala/com/daml/ledger/rxjava/grpc/CommandClientImplDeprecatedTest.scala

```diff
@@ -0,0 +1,298 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.rxjava.grpc
+
+import java.time.{Duration, Instant}
+import java.util.{Optional, UUID}
+import java.util.concurrent.TimeUnit
+
+import com.daml.ledger.javaapi.data.{Command, CreateCommand, DamlRecord, Identifier}
+import com.daml.ledger.rxjava._
+import com.daml.ledger.rxjava.grpc.helpers.{DataLayerHelpers, LedgerServices, TestConfiguration}
+import com.daml.ledger.api.auth.{AuthService, AuthServiceWildcard}
+import com.daml.ledger.api.v1.command_service.{
+  SubmitAndWaitForTransactionIdResponse,
+  SubmitAndWaitForTransactionResponse,
+  SubmitAndWaitForTransactionTreeResponse,
+}
+import com.google.protobuf.empty.Empty
+import io.reactivex.Single
+import org.scalatest.OptionValues
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
+
+import scala.concurrent.Future
+import scala.jdk.CollectionConverters._
+
+//TODO: Should be removed in rxjava 3 copy #15180
+@Deprecated
+class CommandClientImplDeprecatedTest
```

Comment:
These deprecated tests are causing the build to fail because deprecation warnings are converted to errors. Is there a way to skip that check for these tests or better still, do we need to keep the tests of the deprecated functionality?

Reply:
Deprecation only makes sense to me if it is user facing. Just remove the tests, or remove the deprecated annotation and add a TODO pointing to an issue.